int32[5] angle_cmd
int32 msg_counter